### ISSUE TITLE
Support for "Tab" in mobile

### ISSUE THREAD

**Issue by rodrigocipriani** [hackmdio/codimd#1184]
Would be great in case that in menu top bar options, at least in mobile, it have a button to "tab" the content. Because mobile keyboard didn't have it.

What you think about this folks?

**Comment by HerHde** [comment]
You can use spaces instead of tabs. The footer bar in the desktop version has a config option for it, including the number of spaces per tab.
Another button would only on mobile devices wouldn't be too bad, though, if it doesn't get too crowded.

**Comment by rodrigocipriani** [comment]
yeah, I know that it´s possible to use space, but this is not so easy to use as tab. I think maybe a new button only at mobile should be good..
I can try to work on that.

What do u think about?
